manager/build.gradle.kts

```diff
@@ -89,6 +89,24 @@ fun Project.configureBaseExtension() {
             signingConfigs["debug"]
         }
 
+        if (signFile.canRead()) {
+            val prop = Properties()
+            prop.load(signFile.inputStream())
+            var APPLICATION_ID = prop.getProperty("APPLICATION_ID")
+            var APPLICATION_NAME = prop.getProperty("APPLICATION_ID")
+            if (APPLICATION_ID.isBlank()){
+                APPLICATION_ID = "me.weishu.kernelsu"
+            }
+            if (APPLICATION_NAME.isBlank()){
+                APPLICATION_NAME = "KernelSU"
+            }
+            defaultConfig.applicationId = prop.getProperty("APPLICATION_ID")
```

Comment:
it should be `defaultConfig.applicationId = APPLICATION_ID`

Reply:
Sorry, I was careless